internal/provider/generators/shared/codegen/emitter.go

```diff
@@ -603,11 +657,9 @@ func (e Emitter) emitAttribute(tfType string, attributeNameMap map[string]string
 				if len(patternProperty.PatternProperties) > 0 {
 					return features, unsupportedTypeError(path, "key-value map of key-value map")
 				}
-
```

Comment:
See https://github.com/hashicorp/terraform-provider-awscc/pull/2368#pullrequestreview-3047456368.
Please restore the previous spacing. Thanks.